app/views/departments/edit.html.slim

```diff
@@ -4,6 +4,5 @@
 
 == render 'form'
 
-.row
-  .col-xs-12
-    = link_to t(:back), departments_path, class: 'btn btn-default'
\ No newline at end of file
+= form_navigation_btn :show
```

Comment:
It is there due to the fact that it hase been build upon a state you havent removed that button. And now it is here because of rebasing